sosia/utils/logger.py

```diff
@@ -0,0 +1,20 @@
+"""Module to configure the logging for the package."""
+import logging
+from pathlib import Path
+
+_cache_folder = Path.home() / '.cache' / 'sosia/'
```

Comment:
Can we name it `LOG_FILE` please and include the full file name? Will make it easier for me to put all filepaths in a separate file later on.